contrib/etime/README.md

```diff
@@ -7,31 +7,44 @@ The Opentenbase ETime Extension(eTime) provides power to record query(**select s
 ## How to use
 Move this reposity into `${SOURCECODE_PATH}/contri` directory directly.  And under this directory, run `make -sj && make install` to install.
 
-In psql session, you should run `create extension etime;` to create etime extension and `load '$libdir/etime';` to load dynamic file firstly.
+In psql session, you should run `create extension etime;` to create etime extension and `load '$libdir/etime';` to load dynamic file firstly.  And you can set name of table which will be written by record.
 
-After that, you run query and see execution time in log file.
+After that, you run query and see execution time in table you pre-set.
 
 ## Settings
 **min_value**
-You can set the threshold in mircosecond.  the query execution time above that will be recorded in log file.
+You can set the threshold in mircosecond.  the query execution time above that will be recorded in table.
 
 In Opentenbase psql session, You run `set etime.min_value=1000` and the execution time below 1000us will be recorded.
 
 The default is: `0`.
 
+**tablename**
+You can set the table name.  the query execution time above will be recorded in this table.
+
+In Opentenbase psql session, You run `set etime.tablename="etime_t"` and record will be written in the table named `etime_t`.
+
+Last but not least, The table must be have `specific schema` like: `(sql text, time bigint);`.
+
+The default is: `NULL`.
+
 ## Example
 *SQL*
 ```
 set etime.min_value=1000;
 
 create table foo(id bigint, str text) distribute by shard(id);
 insert into foo values(1, 'tencent'), (2, 'shenzhen');
+insert into foo select 1, 'asdasd';
 select * from foo;
 ```
-*Key Log Output*
+*Table Output*
 ```
-SQL: (select * from foo;) Execution Time: 0s, 1045us
+                 sql                 | time
```

Comment:
I can almost understand your implementation, but it seems that the example you supply here is not that correct and clear. Is the statement "etime.tablename = xxx" lost ? And Why the statement "select * from foo;" turns out the slow queries results? By the way, maybe you could add some test cases like other plugins if convenient, which takes efforts but would help your implementation be more maintainable. Anyway, the current implementation is already a good job!